### DISCUSSION TITLE
Switchbot loock

### DISCUSSION THREAD

**Discussion by maxtrin87** [Blackymas/NSPanel_HA_Blueprint#375] [Q&A]
Hello, Has enyone manage to get the switchbot loock in the nspanel after the esphome flash ?

**Comment by Blackymas** [comment]
I haven't yet - what exactly do you need for it?

A lot can be built via helper and then put the helper on a button

**Reply by maxtrin87** [reply]
I want to be able to lock and unlock, so just like a light button.
Also strugling with switchboot curtain road i have 2 and want them to be just one button on the nspanel.

**Reply by Blackymas** [reply]
ok that will then be difficult to do with a button but in the next update 3.2 may come a function that could help you

**Reply by Maxtr1n** [reply]
Sounds great is there any where we can read about the upcoming update? 
And also will it be easy to do the update?

**Reply by Blackymas** [reply]
https://github.com/Blackymas/NSPanel_HA_Blueprint/discussions/384

**Comment by edwardtfn** [comment]
- #1178 
- #606

**Comment by edwardtfn** [comment]
I've implemented this in `dev`, however I cannot test it.
If someone can test it and everything goes well this should be part of v4.3 to be released very soon.
